wait_for_it/wait_for_it.py

```diff
@@ -152,6 +152,8 @@ def tell_neutral(cls, message):
 
 class _ConnectionJobReporter:
     def __init__(self, host, port, timeout):
+        if not host:
```

Comment:
This should probably be `if host is None` with regard to https://www.python.org/dev/peps/pep-0008/#programming-recommendations ?

Reply:
Updated to `if host is None`.